data_safe_haven/infrastructure/components/dynamic/blob_container_acl.py

```diff
@@ -97,6 +97,10 @@ def diff(
         id(id_)
         return self.partial_diff(old_props, new_props)
 
+    def refresh(self, props: dict[str, Any]) -> dict[str, Any]:
+        """TODO: check whether ACLs have changed"""
```

Comment:
Should we open an issue for this?
I guess it is a known limitation of our implementation?

Reply:
I think this code shouldn't be too hard to write but it's not relevant to this PR.